ddtrace/internal/datadog/profiling/stack_v2/src/sampler.cpp

```diff
@@ -7,7 +7,7 @@
 using namespace Datadog;
 
 void
-Sampler::sampling_thread(unsigned long seq_num)
+Sampler::sampling_thread(uint64_t seq_num)
```

Comment:
⛏️ `const uint64_t`

Reply:
Absolutely, fixed.  I'll run this through my cppcheck harness.